src/Api/FileUploads.php

```diff
@@ -39,9 +39,9 @@ public function baseUrl()
      * @param  string  $purpose
      * @return array
      */
-    public function create($file, $purpose)
+    public function create($file, $purpose, $account = null)
```

Comment:
Sorry, have very little time to spare here at work.
Feel free to scrap it and rebuild as you see fit. I've applied my changes locally.